README.md

```diff
@@ -254,6 +254,18 @@ spec:
 
 In the above case, `myapp` and `myapp2` containers will be instrumented, `myapp3` will not.
 
+#### Override Service Name
+
+The service name for containers goes in the following priority list
```

Comment:
missing `:` at the end

Reply:
We should provide some more information about how the service name is determined e.g.: 

> The operator automatically determines the most suitable service name from the Kubernetes object names. However the service name can be overridden `TBD` annotation in the pod specification. Follows the precedence order for determining the service name: